### PR TITLE
Added new sections - portfolio, footer.

### PR DESCRIPTION
**Pull request by OlegAdamovich**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -290,3 +308,209 @@
...
+    justify-content: space-between;
+    align-items: center;
+    margin: 43px 0 0;
+}
+
+.footer-info-social-icons {
+    display: flex;
+}
+
+.footer-info-social-icon-item-bg {
```
Comment (mhryshkin) [inline style.css:507]: is it one big block?

File: style.css [hunk 2]
```diff
@@ -290,3 +308,209 @@
...
+
+.portfolio__items {
+}
+
+.portfolio_item {
+    display: flex;
+    margin-bottom: 56px;
+}
+
+.portfolio_date {
```
Comment (mhryshkin) [inline style.css:351]: is data a modifier?

File: style.css [hunk 3]
```diff
@@ -290,3 +308,209 @@
...
+}
+
+.portfolio__content {
+    display: flex;
+}
+
+.portfolio__items {
+}
+
+.portfolio_item {
```
Comment (mhryshkin) [inline style.css:346]: is an item a modifier?

File: style.css [hunk 4]
```diff
@@ -290,3 +308,209 @@
...
+    color: #FFF;
+    font-size: 17px;
+    line-height: 28px;
+    font-weight: 700;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.portfolio__item-description {
```
Comment (mhryshkin) [inline style.css:386]: Do we need it?

File: style.css [hunk 5]
```diff
@@ -290,3 +308,209 @@
...
+    justify-content: center;
+    align-items: center;
+    height: 106px;
+    width: 106px;
+    border-radius: 50%;
+    background: #FFF;
+    margin-left: 41px;
+}
+
+.portfolio__item-review-img {
```
Comment (mhryshkin) [inline style.css:457]: Do we need it?

File: style.css [hunk 6]
```diff
@@ -290,3 +308,203 @@
...
+    display: flex;
+    margin-bottom: 56px;
+}
+
+.portfolio__date {
+    width: 70px;
+    margin-right: 29px;
+}
+
+.portfolio__date-top {
```
Comment (mhryshkin) [inline style.css:356]: @OlegAdamovich looks like `portfolio__date` has pretty much in common with `.portfolio__date-bottom `, any ideas how to make it better?

File: style.css [hunk 7]
```diff
@@ -290,3 +308,203 @@
...
+}
+
+.footer__info {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 43px 0 0;
+}
+
+.footer__info-social-icons {
```
Comment (mhryshkin) [inline style.css:497]: `footer-info__...` ?

File: style.css (hunk 3 above)
Comment (OlegAdamovich) [reply style.css]: Fixed.

File: style.css (hunk 6 above)
Comment (OlegAdamovich) [reply style.css:356]: Сreate a modifier `portfolio__date_bottom`.

File: style.css (hunk 7 above)
Comment (OlegAdamovich) [reply style.css:497]: Fixed.

**Comment by mhryshkin** [comment]
LGTM

**Review by mhryshkin — APPROVED** [review]

**Pull request merged**